osu.Game/Rulesets/Mods/ModHidden.cs

```diff
@@ -32,6 +34,26 @@ public virtual void ApplyToDrawableHitObjects(IEnumerable<DrawableHitObject> dra
                 d.ApplyCustomUpdateState += ApplyHiddenState;
         }
 
+        public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
+        {
+            // Default value of ScoreProcessor's Rank in Hidden Mod should bes SS+
```

Comment:
```suggestion
            // Default value of <see cref="ScoreProcessor"/>'s Rank in Hidden Mod should be SS+
```

Reply:
That's not an xmldoc..? + that's just a comment telling that the default value for `ScoreProcessor.Rank` should change to XH if hidden mod was selected.